backend/tournesol/tests/test_api_unconnected_entities.py

```diff
@@ -140,10 +148,15 @@ def test_no_link_should_return_all(self):
         self.assertEqual(response.data["count"], 3)
 
 
-class AdvancedAllConnectedTest(TestCase):
+class SingleGraphNonReducibleDistanceTestCase(TestCase):
     """
-    TestCase for the unconnected entities API. 
-    In this Advanced test, all video are not necessary connecter to the user
+    A test case of the unconnected entities API.
+
+    Here, all entities are not necessarily connected together to by the tested
+    user, yet forming a single graph.
```

Comment:
What you mean by "single graph" is "connected graph"

Reply:
I see, I'll update the comment.